pkg/util/ptr.go

```diff
@@ -0,0 +1,5 @@
+package util
+
+func Ptr[T any](x T) *T {
+	return &x
+}
```

Comment:
Let's use `github.com/samber/lo` instead? Previous discussion here: https://github.com/neondatabase/autoscaling/pull/895#discussion_r1576977602

Reply:
New PR here: #946

Reply:
Fixed in 71558992d633a760c8f5e04535ce5869118df2b0